src/formats/webm.moon

```diff
@@ -19,14 +19,18 @@ class WebmVP8 extends Format
 		colormatrix = mp.get_property_native("video-params/colormatrix")
 		if colormatrixFilter[colormatrix]
 			append(ret, {
-				"lavfi-colormatrix=#{colormatrixFilter[colormatrix]}:bt601"
+				MpvFilter("lavfi-colormatrix",
+					"@0": colormatrixFilter[colormatrix],
+					"@1": "bt601")
```

Comment:
Should pass a table as a argument:
```moon
MpvFilter("lavfi-colormatrix",
		{ "@0": colormatrixFilter[colormatrix],
		  "@1": "bt601" })
```

If you decide to drop the positional parameters, `src` and `dst` are the option names, respectively.

Reply:
My bad... Not at a computer for a while so can't fix this right now. If you want to fix it then you can go ahead and merge it whenever, otherwise I'd have to do this tomorrow.

Reply:
All right, I'll work on it when I have time.